### DISCUSSION TITLE
How to select text between dashes, without selecting the whole word

### DISCUSSION THREAD

**Discussion by dimitrisnl** [zed-industries/zed#12996] [Feature Requests]
Maybe this is easier to understand. I want to select `white` without selecting the whole `bg-white`.

Usually out of habit, I select something, and write my replacement, only to realize that instead of `bg-red`, I've overwritten the whole text.

![CleanShot 2024-06-13 at 20 20 33](https://github.com/zed-industries/zed/assets/4951004/e99df0da-b812-4074-93d2-422d7eb9ba78)

**Comment by dimitrisnl** [comment]
Bump; should I open this as an issue?

**Comment by dimitrisnl** [comment]
FYI I have open a ticket https://github.com/zed-industries/zed/issues/15606 for anyone who wants to chime in

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
